b3sum/src/main.rs

```diff
@@ -42,6 +43,11 @@ fn clap_parse_argv() -> clap::ArgMatches<'static> {
                 .long(NO_NAMES_ARG)
                 .help("Omits filenames in the output"),
         )
+        .arg(
+            Arg::with_name(RAW_ARG)
+                .long(RAW_ARG)
+                .help("Raw output without hexidecimal encoding. \nOnly one filename may be provided and no filenames will be displayed."),
```

Comment:
Interesting, I didn't know Clap did the right thing with newlines in these strings. Very useful!